examples/src/main/java/UpdateNftsMetadataExample.java

```diff
@@ -98,36 +102,40 @@ private void updateNftsMetadata(TokenCreateTransaction tokenCreateTransaction) t
         });
 
         var tokenMintResponse = tokenMintTransaction.execute(client);
+        
         // Get receipt for mint token transaction
         var tokenMintReceipt = tokenMintResponse.getReceipt(client);
         System.out.println("Status of token mint transaction: " + tokenMintReceipt.status);
 
         var nftSerials = tokenMintReceipt.serials;
-        // Check that metadata was set correctly
+        // Check that metadata on the NFT was set correctly
         getMetadataList(client, tokenId, nftSerials).forEach(metadata -> {
             System.out.println("Metadata after mint: " + Arrays.toString(metadata));
         });
 
+        //Create an account to send the NFT to
         var accountCreateTransaction = new AccountCreateTransaction()
             .setKey(OPERATOR_KEY)
             .setMaxAutomaticTokenAssociations(10)
             .execute(client);
 
         var newAccountId = accountCreateTransaction.getReceipt(client).accountId;
 
-        new TokenAssociateTransaction()
-            .setAccountId(newAccountId)
-            .setTokenIds(Collections.singletonList(tokenId))
-            .freezeWith(client)
-            .sign(OPERATOR_KEY)
-            .execute(client)
-            .getReceipt(client);
+        //Associate the NFT to the account only if the account does not have any automatic token association slots open. When we created the account in the previous step we set automatic token associations to 10 so I should have 10 slots open to receive a token
+        //new TokenAssociateTransaction()
+            //.setAccountId(newAccountId)
+            //.setTokenIds(Collections.singletonList(tokenId))
+            //.freezeWith(client)
+            //.sign(OPERATOR_KEY)
+            //.execute(client)
+            //.getReceipt(client);
```

Comment:
It's not a good practice to push a commented code. I would recommend to delete these lines.